tests/src/Functional/NewsPageTest.php

```diff
@@ -77,24 +84,95 @@ public function testNewsFields() {
   }
 
   /**
-   * Pathauto and breadcrumbs.
+   * News article, newsroom, featured news.
    */
-  public function testNewsPaths() {
-    $this->createNode([
+  public function testNewsPages() {
+    $body = $this->randomMachineName(64);
+    $newsArticle = $this->createNode([
       'title' => 'News article 1',
+      'body' => $body,
       'type' => 'localgov_news_article',
       'status' => NodeInterface::PUBLISHED,
     ]);
 
-    $this->drupalGet('news');
+    $this->drupalGet('news/' . date('Y') . '/news-article-1');
     $this->assertText('News article 1');
+    $this->assertText($body);
+
+    $newsroom = $this->createNode([
+      'title' => 'News',
+      'type' => 'localgov_newsroom',
+      'path' => [
+        'alias' => '/news',
+      ],
+      'status' => NodeInterface::PUBLISHED,
+    ]);
 
-    $this->drupalGet('news/' . date('Y') . '/news-article-1');
+    $this->drupalGet('news');
     $this->assertText('News article 1');
 
-    $trail = ['' => 'Home'];
-    $trail += ['news' => 'News'];
-    $this->assertBreadcrumb(NULL, $trail);
+    // Add News article 1 to the featured news block.
+    $newsroom->set('localgov_newsroom_featured', $newsArticle->id());
+    $newsroom->save();
+    drupal_flush_all_caches();
```

Comment:
Is there a reason for the flush all caches? Be good to have a look at the cache tags on that block and remove this. Feels like updating a news featured block shouldn't require a cahce clear, and if removing this causes the test to fail, that indicated something that needs fixing.

Reply:
It fails without clearing caches. I've no experience with cache tags but will take a look, good opportunity to learn.

Reply:
Ok, we can merge for now, however I'd look at the localgov services for how to do cache tags and build the test without fluch caches. https://github.com/localgovdrupal/localgov_services/blob/7ce43ae55eadfb286405ef144b50ecddaf888d72/src/Plugin/Block/ServicesBlockBase.php#L92

Reply:
Thanks @andybroomfield, I'll add an issue and have a look at that.